src/chunk.c

```diff
@@ -3195,8 +3195,8 @@ ts_chunks_in(PG_FUNCTION_ARGS)
 	pg_unreachable();
 }
 
-/* Check if this chunk can be compressed, that it is not dropped and has not
- * already been compressed. */
+/* Check if this chunk can be compressed or reordered, that it is not dropped
```

Comment:
could you revert the comment change as well? We are only checking if it can be compressed